src/test/java/seedu/ccacommander/logic/LogicManagerTest.java

```diff
@@ -155,10 +155,10 @@ private void assertCommandFailure(String inputCommand, Class<? extends Throwable
     private void assertCommandFailureForExceptionFromStorage(IOException e, String expectedMessage) {
```

Comment:
At line 161, `addressBook` is still used as the name of the parameter for the method `saveCcaCommander`.

Reply:
resolved